lib/bb/tests/event.py

```diff
@@ -0,0 +1,966 @@
+#
+# BitBake Tests for the Event implementation (event.py)
+#
+# Copyright (C) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+import collections
+import importlib
+import logging
+import pickle
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Consider possible security implications associated with pickle module. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b403-import-pickle